compiler/rustc_llvm/llvm-wrapper/Linker.cpp

```diff
@@ -1,3 +1,4 @@
+#include "SuppressLLVMWarnings.h"
```

Comment:
drive-by comment: shouldn't this be unnecessary since this file also includes `LLVMWrapper.h`?

Reply:
It does, but only after already including several other header files.  The `llvm-c` headers are fine, but `llvm/Linker/Linker.h` is a C++ header, and so could have the same issues.